docker-compose.staging.yml

```diff
@@ -8,10 +8,10 @@ version: "3"
 services:
   web:
     restart: always
-    hostname: "$HOSTNAME"
+    hostname: ${HOSTNAME:-$HOST}
     environment:
-      - PYENV_VERSION=${PYENV_VERSION:-"3.8.6"}
-      - OL_CONFIG=/olsystem/etc/openlibrary.yml
       - GUNICORN_OPTS= --workers 4 --timeout 180
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
+      - PYENV_VERSION=${PYENV_VERSION:-"3.8.6"}
```

Comment:
```suggestion
      - PYENV_VERSION=3.8.6
```

Reply:
This modification removes our ability to try Python 2.7.6, 3.8.6, and 3.9.0 from the command line.
`PYENV_VERSION=3.9.0 sudo docker-compose exec web bash` followed by `echo $PYENV_VERSION` will always print `3.8.6` and not the intended `3.9.0`.